test/hdfs.c

```diff
@@ -28,6 +28,9 @@
 #define HDFS_TEST_DEBUG        0
 #define HDFS_TEST_MAX_BUF_SIZE 256
 
+#define S3_TEST_RESOURCE_TEXT_RESTRICTED_SIZE 5582655
+#define S3_TEST_RESOURCE_TEXT_PUBLIC_SIZE     6589
```

Comment:
Seems like the changes to this file may have been unintentional. Are these files in S3 considering that this is the HDFS test? The values for the macros also don't seem to match up with the values they're replacing below.